src/components/input-fields/simple-input/SimpleInput.tsx

```diff
@@ -141,23 +149,43 @@ export const SimpleInput: FC<ISimpleInput> = ({
             {required && !inputRef?.current?.value && !label && (!errors || errors.length === 0) && (
               <SSimpleInputRequiredIndicator />
             )}
-            <SSimpleInput
-              width={inputWidth}
-              error={state === "error"}
-              locked={state === "locked"}
-              onChange={(e) => {
-                onChange?.(e);
-              }}
-              required={required}
-              ref={inputRef}
-              onFocus={handleFocus}
-              onBlur={handleBlur}
-              value={value || ""}
-              placeholder={placeholder}
-              disabled={disabled}
-              onKeyUp={onKeyUp}
-              type={type}
-            />
+            {multiline ? (
```

Comment:
Can we move the conditional outside the render and have a single render so we can avoid duplicating code?